src/screens/Exchange/ExchangeTextInput.js

```diff
@@ -0,0 +1,102 @@
+// @flow
```

Comment:
I see that this component is planned to be used in liquidity pool feature. What do you think about moving it outside of Exchange folder so it would feel more reusable (independent)?